Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Building this image will include the running of all benchmarks in a "cold
+# cache" state, and thus will take quite a bit of time (roughly 30-60 minutes,
+# with Ganache excluded).  The cache will be retained in the built image, such
+# that running the image will execute benchmarks with a "warm cache". For
+# example:
+#
+#   docker --build-arg ALCHEMY_API_KEY=... -t sim .
+#   docker run sim benchmark-hardhat
+#
+# BEWARE: This image should not be published because it will contain traces of
+# your Alchemy API key. (See https://stackoverflow.com/a/40762010/406289)
+# Future work can take advantage of Docker's support for secrets, but
+# unless/until there's a need to publish this image, that would be overkill.
+
+FROM debian
+
+ARG ALCHEMY_API_KEY
```

Comment:
Can we update this to the full `ETH_RPC_URL` instead of just overwriting the API key portion? In the `.env.template` I use alchemy as an example, but in general I prefer specifying the full RPC URL as the env var so it's easier to use any node provider / a local node

Reply:
Addressed in https://github.com/mds1/convex-shutdown-simulation/pull/5/commits/e3d61d6691bfcd3d72907714ff00a226e648dd75